upf/upf.api

```diff
@@ -237,3 +237,67 @@ define upf_nwi_dump {
   u32 client_index;
   u32 context;
 };
+
+enum intf_type : u8 {
+  S_INTF_ACCESS = 0,
+  S_INTF_CORE = 1,
+  S_INTF_SGI_LAN = 2,
+  S_INTF_CP = 3,
+  S_INTF_INVALID = 255,
+};
+
+autoreply define upf_gtpu_endpoint_add_del {
+  u32 client_index;
+  u32 context;
+  u8 is_add;
+  vl_api_ip4_address_t ip4;
+  vl_api_ip6_address_t ip6;
```

Comment:
The API implementation is the same as CLI implementation. So, if original CLI code does have dedicated IP4/IP6 entities, dedicated API should also have it.

Reply:
I think it would be better to make UPG run in Rel.16 mode only and remove the obsolete User Plane IP Resource Information support altogether.

However, that should first be run by Product Management.